account_tax_one_vat/models/one_vat_mixin.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, models
+from odoo.exceptions import ValidationError
+
+
+class OneVatMixin(models.AbstractModel):
+    _name = "one.vat.mixin"
+    _description = (
+        "Abstract the methods needed to ensure only one VAT tax is " "selected."
+    )
+
+    def _check_only_one_vat_tax_field(self, field_name):
+        vat_group = self.env.ref("account_tax_one_vat.vat_tax_group")
+        for rec in self:
+            if getattr(rec.company_id, "account_tax_one_vat", False):
```

Comment:
`getattr` is useless... you can simply write `if rec.company_id.account_tax_one_vat`